### PR TITLE
Download azure blob

### PR DESCRIPTION
**Pull request by MallikharjunaThota123**
Successful Runs with model source 
1. Azure blob 
Run ID:- https://ml.azure.com/runs/elated_nut_r3hqz3qt43?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourcegroups/shared-finetuning-rg/workspaces/v-mathota&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

image

2. GIT
Run ID:- https://ml.azure.com/experiments/id/25167191-e537-41f2-8e32-cc80e57b8a25/runs/cff7a79a-10e7-4894-9f15-95e039710b59?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-mathota&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

image

Azure Blob versions: 
Download Component:- https://ml.azure.com/registries/azureml-preview-test1/components/download_model?tid=72f988bf-86f1-41af-91ab-2d7cd011db47

Download component version: - 0.2.10.blobtest1

Import component:- https://ml.azure.com/registries/azureml-preview-test1/components/import_model?tid=72f988bf-86f1-41af-91ab-2d7cd011db47

import component version: - 0.2.8.test_blob_storage1

### CODE REVIEW

File: assets/training/model_management/src/azureml/model/mgmt/downloader/downloader.py [hunk 1]
```diff
@@ -64,15 +64,19 @@ def __init__(self, model_uri: str, download_dir: Path):
...
             # TODO: Handle error case correctly, since azcopy exits with 0 exit code, even in case of error.
             # https://github.com/Azure/azureml-assets/issues/283
             exit_code, stdout = run_command(download_cmd)
             if exit_code != 0:
                 raise AzureMLException._with_error(
                     AzureMLError.create(BlobStorageDownloadError, uri=self._model_uri, error=stdout)
                 )
-
+            # Log the contents of the downloaded directory
+            downloaded_files = list(Path(self._download_dir).rglob("*"))
```
Comment (vikasagrawal-ms) [inline assets/training/model_management/src/azureml/model/mgmt/downloader/downloader.py:78]: remove these.

File: assets/training/model_management/src/azureml/model/mgmt/downloader/downloader.py [hunk 2]
```diff
@@ -64,15 +64,19 @@ def __init__(self, model_uri: str, download_dir: Path):
     @log_execution_time
     def _download(self):
         try:
-            download_cmd = f"azcopy cp --recursive=true '{self._model_uri}' {self._download_dir}"
+            logger.info(f"self._model_uri: {self._model_uri}")
```
Comment (vikasagrawal-ms) [inline assets/training/model_management/src/azureml/model/mgmt/downloader/downloader.py:67]: logging can also be removed

**Review by vikasagrawal-ms — COMMENTED** [review]
Add description and add version changes for download and import component

File: assets/training/model_management/src/azureml/model/mgmt/downloader/downloader.py (hunk 1 above)
Comment (MallikharjunaThota123) [reply assets/training/model_management/src/azureml/model/mgmt/downloader/downloader.py]: removed the lines

**Review by vikasagrawal-ms — APPROVED** [review]

**Pull request merged**